catalog/ui/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -139,6 +147,11 @@ define([
             }
             return cqlPolygon;
         },
+        polygonToCQLMultiPolygon: function(model) {
+            return model.map(function (polygon) {
+                return this.polygonToCQLPolygon(polygon);
+            }.bind(this));
```

Comment:
I believe that this can be simplified to `return model.map(this.polygonToCQLPolygon);`.

Reply:
Changed